packages/generator/ni_measurement_plugin_sdk_generator/plugin/__main__.py

```diff
@@ -1,5 +1,5 @@
 """"Creates a measurement through use of template.py."""
```

Comment:
Update the docstring.
```suggestion
""""Creates a measurement through use of __init__.py."""
```